drivers/pwm/pwm_stm32.c

```diff
@@ -313,16 +318,21 @@ static int pwm_stm32_init(const struct device *dev)
 	init.CounterMode = LL_TIM_COUNTERMODE_UP;
 	init.Autoreload = 0u;
 	init.ClockDivision = LL_TIM_CLOCKDIVISION_DIV1;
+#ifndef CONFIG_SOC_SERIES_STM32L0X
```

Comment:
I think `RepetitionCounter` could actually be removed, `LL_TIM_StructInit` takes care of initializing the field if it's available to zero.

Reply:
You are totally correct. Double-checked the init function for all MCUs and removed it.